tests/orquestra/quantum/wavefunction_test.py

```diff
@@ -374,3 +375,14 @@ def test_sample_from_wavefunction_fails_for_invalid_n_samples(n_samples):
     wavefunction = Wavefunction(amplitudes)
     with pytest.raises(ValueError):
         sample_from_wavefunction(wavefunction, n_samples)
+
+
+def test_sample_from_wavefunction_is_efficient_for_high_n_samples():
```

Comment:
This test shouldn't:
a) be here
b) look like this :)
Please see the [performance tests repo](https://github.com/zapatacomputing/orquestra-performance-tests) :)

Reply:
Got rid of it!